docs/en/common/core/coffee-grpc.adoc

```diff
@@ -96,6 +96,19 @@ coffee:
 ----
 <1> if `true`, then `coffee.grpc.server.threadpool.default` is ignored.
 
+In case `ManagedExecutorService` thread pool handling an additional Wildfly configuration is required.
+
+.Wildfly gRPC managed executor service configuration
```

Comment:
Quarkus hasznalatra is gondolni kell, ott hogyan lehet ezt a cofee grpc modult bekotni bekotni? Valamit talaltam, de nem egyertelmu az itteni megoldashoz: https://github.com/quarkusio/quarkus/discussions/39096

Reply:
Jogos, viszont el kell döntenünk melyik iranyba megyünk quarkussal: https://github.com/i-Cell-Mobilsoft-Open-Source/quarkus-sampler/blob/main/qs-services/qs-grpc-service/src/main/java/hu/icellmobilsoft/quarkus/sampler/grpc/action/SampleGrpcService.java#L35

A tesztjeink lefutottak rajta ugy hogy a coffee-bol a generalast is használhatjuk, viszont az annotacók a quarkus-ból jönnek.

Reply:
Jelenleg ez a "Starter" csak WF rendszerbn tud mukodni, Quarkus-nak van sajat GRPC "starter", tehat akkor ez a managedExecutor ezen nem valtoztat. Viszont vannak interceptorok amelyek a GRPC server-kliens kommunikaciot stardardizaljak coffee szinten, azokat ki fog kelleni emelni kulon csomagokba hogy a quarkusban is hasznalhato legyen. Ezt majd akkor masik MR-ben kellene megtenni